### PR TITLE
op-mode: T3384: Support UDP bandwidth testing

### PR DESCRIPTION
**Pull request by jack9603301**
## Change Summary

Support UDP bandwidth testing

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

https://phabricator.vyos.net/T3384

## Component(s) name

op-mode

## Proposed changes

Support UDP bandwidth testing

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by c-po** [comment]
Listening on UDP makes less sense as the initiator command runs on TCP only.

**Comment by jack9603301** [comment]
> Listening on UDP makes less sense as the initiator command runs on TCP only.

Has supported

File: op-mode-definitions/monitor-bandwidth-test.xml.in [hunk 1]
```diff
@@ -7,21 +7,50 @@
           <help>Initiate or wait for bandwidth test</help>
         </properties>
         <children>
-          <leafNode name="accept">
+          <node name="accept">
```
Comment (c-po) [inline op-mode-definitions/monitor-bandwidth-test.xml.in:10]: The old `accept` node simply called: `<command>/usr/bin/iperf -V -s</command>`, can we make this backwards compatible and behave in the same way?

**Review by c-po — CHANGES_REQUESTED** [review]

File: op-mode-definitions/monitor-bandwidth-test.xml.in (hunk 1 above)
Comment (jack9603301) [reply op-mode-definitions/monitor-bandwidth-test.xml.in:10]: Please check if my amendment is correct. I'm not sure if I understand your statement?

**Pull request merged**